gokart/slack/slack_config.py

```diff
@@ -5,3 +5,6 @@ class SlackConfig(luigi.Config):
     token_name = luigi.Parameter(default='SLACK_TOKEN', description='slack token environment variable.')
     channel = luigi.Parameter(default='', description='channel name for notification.')
     to_user = luigi.Parameter(default='', description='Optional; user name who is supposed to be mentioned.')
+    send_tree_info = luigi.BoolParameter(default=True,
```

Comment:
When `default=True`,  `send_tree_info` can not set False in command line.

Reply:
Yeah, I see your point. But, if `default=False`, this updating is not compatible for backward versions.
I think better that this parameter is set to `False` in script file including `gokart.run()`  or `config.ini` than backward compatible is lost.

When `default=False` is adopted, to retain backward compatible, the code style maybe become ugly... (ex. `send_tree_info` is named `do_not_send_tree_info`.)

Reply:
@Ryusuketa I think you may not care about "backward compatible".

Reply:
@nihsiba OK. I set `False` to default value.